spec/spec_helper.rb

```diff
@@ -56,6 +57,7 @@
   config.use_transactional_fixtures = false
   config.example_status_persistence_file_path = "tmp/failed_examples.txt"
 
+  config.include Devise::Test::ControllerHelpers, type: :controller
```

Comment:
The controller inheritance has changed from `Spree::Api::BaseController` to `Spree::BaseController`. This was done because we need to generate the client token for braintree with the customer id. To get that we need to know the user logged in.

This change resulted in spec failures:
```
Devise::MissingWarden:
       Devise could not find the `Warden::Proxy` instance on your request environment.
```